packages/utils/hooks/useDebounce/index.ts

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
I was having issues with an import of a utils function in a server side component because of these guys that use useState or useEffect, so I added the client directive.

Reply:
you mean while importing them on the template project?

Reply:
Yes, importing them on the template project in server components. Because we do this barrel export thing in the main index.ts of the package @Janekk , and if you don't specify the exact absolute path, nextjs ends up trying to import client things on the server side.